### ISSUE TITLE
[DBG] prompt not displayed when breakpoint hit unless Enter is pressed

### ISSUE THREAD

**Issue by sba923** [PowerShell/vscode-powershell#1850]
### Issue Description

When I debug a PowerShell script and hit a breakpoint, the [DBG] prompt is not displayed immediately, but only after I hit Enter.

Here's what the TERMINAL pane shows when I hit F5 and the first breakpoint is hit:

````
[… output from various profile scripts]
PS ~/powershell> s:\powershell\df.ps1
````

When I then hit Enter this changes to:

````
[… output from various profile scripts]
PS ~/powershell> s:\powershell\df.ps1
[DBG]: PS ~/powershell>
````

#### Expected Behaviour

When the breakpoint is hit, the [DBG] prompt should be displayed right away.

#### Actual Behaviour

The [DBG] prompt is not displayed unless Enter is pressed.

### Attached Log
[PSESlogs201904060747.zip](https://github.com/PowerShell/vscode-powershell/files/3050199/PSESlogs201904060747.zip)
s

### System Details
System Details Output
```
### VSCode version: 1.33.0 0dd516dd412d42323fc3464531b1c715d51c4c1a x64

### VSCode extensions:
alefragnani.Bookmarks@10.4.2
anseki.vscode-color@0.4.5
austin.code-gnu-global@0.2.2
DougFinke.vscode-PSStackoverflow@0.0.2
Gruntfuggly.todo-tree@0.0.126
mechatroner.rainbow-csv@1.0.0
ms-vscode.cpptools@0.22.1
ms-vscode.powershell@1.12.0
ms-vscode.powershell-preview@2.0.1
msjsdiag.debugger-for-chrome@4.11.3
nobuhito.printcode@3.0.0
tht13.html-preview-vscode@0.2.5
VisualStudioExptTeam.vscodeintellicode@1.1.5
yzhang.markdown-all-in-one@2.2.0
zovorap.ab-html-formatter@0.0.2


### PSES version: 2.0.0.0

### PowerShell version:

Name                           Value
----                           -----
PSVersion                      5.1.17763.316
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.316
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by sba923** [comment]
This might or might not be related to the following other issue with prompts:

Take the following script, stored at `C:\private_sba\PowerShell\TestPowerShellDebuggerBehaviorWithPrompt.ps1`:

````
Write-Host("WH1")
Write-Host("WH2")
Write-Host("WH3")
````

If you run this via F5 in the Windows PowerShell ISE you get:

````
PS ~> C:\private_sba\PowerShell\TestPowerShellDebuggerBehaviorWithPrompt.ps1
WH1
WH2
WH3
````

But if you do the same in VScode, you get:

````
PS ~/PowerShell/work> WH1private_sba\PowerShell\TestPowerShellDebuggerBehaviorWithPrompt.ps1
WH2
WH3
````

if you're using the "PowerShell Launch Current File" configuration, and

````
PS ~/PowerShell/work> c:\private_sba\PowerShell\TestPowerShellDebuggerBehaviorWithPrompt.ps1

                      WH1
WH2
WH3
````

if you're using the "PowerShell Launch Current File w/Args Prompt" configuration.

Both are incorrect if you ask me. I'll probably open a separate issue for that.

**Comment by SydneyhSmith** [comment]
@sba923 thanks for reporting, both of these issues, the second one does separate, could you please open another issue for that one? We can reproduce this, it looks related to PSReadLine support.
We are having trouble reproducing the first one...which launch configuration are you using?

**Comment by sba923** [comment]
You're welcome.

Opened #1862.

(I've already mentioned which launch configurations I'm using ;-) )

**Comment by sba923** [comment]
I hate to say, but I can repro this with version 2019.11.0 of the PowerShell Preview extension (in conjunction with PSReadLine 2.0.0 beta 5).

I get an additional issue when I hit Enter after the debugger stops at the breakpoint:

![image](https://user-images.githubusercontent.com/12860484/68156479-35663980-ff4c-11e9-90ad-cffae4363418.png)

I will open a new issue for that.

**Comment by rjmholt** [comment]
> I hate to say, but I can repro this with version 2019.11.0 of the PowerShell Preview extension

Yes, this issue remains open because we haven't been able to address it yet.

**Comment by sba923** [comment]
You're right. I should've inferred that from the release notes....

**Comment by rjmholt** [comment]
I looked into this a little bit, but I wasn't able to track down where it happens yet. Given that the workaround is to press enter, I'll have to get back to this one while we solve some of the other issues in the latest preview.

**Comment by TylerLeonhardt** [comment]
I have a fix out for this ^

**Comment by sba923** [comment]
Great! Want me to give it a shot?

**Comment by TylerLeonhardt** [comment]
@sba923 

Keep in mind this is a non-signed non-supported build and only for the sake of verifying that the problem is fixed.

[PowerShell-insiders.vsix.zip](https://github.com/PowerShell/vscode-powershell/files/3837511/PowerShell-insiders.vsix.zip)

Here's the vsix - do you know have to install it?

```
code --install-extension path/to/vsix
```
or
in the GUI on the extensions page.

You may need to uninstall the PowerShell Preview extension in order to install this one as I have not rev'd the version.

**Comment by sba923** [comment]
@TylerLeonhardt I hereby confirm that the fix worked on first try. Great job!

(I was scared at first by the "the extension has been updated to..." notification, fearing that the marketplace version might have overwritten the one from the VSIX...)
